Mimble Wimble.md

```diff
@@ -0,0 +1,124 @@
+
+Mimbleimble was implemented on Litecoin by the use of a feature called Extension Blocks. The implementation launched in May 2022, and made it possible for Litecoin holders to use the new privacy features by moving their Litecoin into these Extension Blocks. These blocks are responsible for hiding the transaction information.
+
+To engage with this privacy feature, users perform a "pegging-in" transaction to move their Litecoin into an Extension Block, using a specific MimbleWimble Extension Block (MWEB) address. These transactions are facilitated by a special transaction known as the HogEx (Hogwarts Express), which processes both the pegging-in of funds to the Extension Block and the pegging-out back to the main, transparent blockchain. The HogEx also manages the Extension Block balance, adjusting it for incoming and outgoing transactions.
+
+While the majority of Litecoin transactions continue to operate on the transparent main blockchain, the introduction of MimbleWimble via Extension Blocks provides a significant step forward in offering privacy options to users.
+
+Implementing MimbleWimble on the Decred blockchain is theoretically possible. Decred governance model and adaptability could facilitate the consideration and potential implementation of MimbleWimble if there was enough community support.
+
+The technical integration of MimbleWimble would require a lot of development effort and a thorough review process to ensure that it aligns with Decred’s security, privacy, and scalability goals. One of the main considerations would be how to integrate MimbleWimble's privacy features with Decred's existing features, such as its governance system, and the Stakeshuffle protocol.
+
+There are also some Mimblewimble drawbacks that need to be taken into consideration: 
+
+Slower Transactions: Networks that use Confidential Transactions, like MimbleWimble, can be slower than others such as Ripple (XRP) and proof-of-stake networks like Solana and Cardano. This is because the data for Confidential Transactions is larger, which can slow things down.
+
+Quantum Computer Risk: MimbleWimble relies on digital signatures to work. This makes it potentially more at risk from future attacks by quantum computers, which might be able to break these digital signatures. Decred has overcome this problem with post-quantum secure mixing on stakeshuffle.
+
+## Moving Forward
+
+Is always interesting to learn about privacy protocols, especially one that is much commented like Mimblewimble. Do you think it could be a good addition to the Decred blockchain? Let your voice be heard in the comment section below!
+
+Sources:
+
+A Formal Analysis of the Mimblewimble Cryptocurrency Protocol 
+
+Non-interactive Mimblewimble transactions revisited
+
+Explaining MimbleWimble: the privacy upgrade to Litecoin
+
```

Comment:
As a final note. This is a very interesting topic and generally reads well. 

My main criticism is it lacks depth and critical thinking. As we move to the new payment structure this will be my next focus for all articles. Things I'd like to see more of:

* Data - If you make a statement like "this is very scalable" it needs to be backed up with data e.g. "this is what the data looks like without ... and here's what it looks like with ..."
* Questioning - Rather than writing a piece as a review or an overview. I'd like each piece to also ask critical thinking questions. For example in the discussion stage, you could say, I'm aiming to answer these 3 questions:
"Is MW more efficient and scalable than typical transactions, if so how?"
"How does MW make transactions more private and what are the trade offs?"
"Is Decred's LN already better than MW or does MW offer some interesting benefits, if so what are they and how do they improve on LN?"